src/ucm/event/event.c

```diff
@@ -334,6 +337,25 @@ void *ucm_sbrk(intptr_t increment)
     return event.sbrk.result;
 }
 
+#if HAVE_CUDA
+cudaError_t ucm_cudaFree(void *addr)
```

Comment:
maybe this should be somewhere in src/ucm/cuda/install.c?

Reply:
I think i have to keep it here, otherwise have to expose ucm_event_enter()/leave()?

Reply:
ok